lld/wasm/Config.h

```diff
@@ -96,24 +101,19 @@ struct Configuration {
   std::optional<std::vector<std::string>> features;
   std::optional<std::vector<std::string>> extraFeatures;
   llvm::SmallVector<uint8_t, 0> buildIdVector;
+};
 
-  // The following config options do not directly correspond to any
-  // particular command line options, and should probably be moved to separate
-  // Ctx struct as in ELF/Config.h
+// The only instance of Configuration struct.
+extern Configuration *config;
 
+struct Ctx {
```

Comment:
How is `Ctx` related to `Configuration`? How do you decide what goes in here vs there?

Reply:
Configuration contains things that correspond to command line flags.  Ctx holds global state.   This is the same as in ELF.

Reply:
Makes sense. Could we maybe improve upon ELF and add a comment to that effect somewhere?

Reply:
Done